per/serializers.py

```diff
@@ -995,13 +1011,14 @@ def validate(self, data):
             )
         return data
 
-
     def create(self, validated_data):
+        if "file" in validated_data:
+            check(validated_data["file"])
```

Comment:
For create, We can ignore "file" in validated_data check as the `file is a required field in PerDocumentUpload.